App.tsx

```
@@ -90,25 +90,25 @@ const BottomNav = ({ onNotesClick }: { onNotesClick: () => void }) => {
-          <Link to="/" className={getLinkClass(isProjects)}>
-              <span className="material-symbols-outlined text-[24px]">grid_view</span>
+          <Link to="/" className={getLinkClass(isProjects)} aria-current={isProjects ? 'page' : undefined}>
+              <span className="material-symbols-outlined text-[24px]" aria-hidden="true">grid_view</span>
               <span className="text-[10px] font-bold">Projects</span>
           </Link>
           <button
               onClick={onNotesClick}
-              className="flex flex-col items-center justify-center w-16 h-full gap-1 text-gray-500 dark:text-gray-400 hover:text-gray-900 dark:hover:text-white transition-colors"
+              className="flex flex-col items-center justify-center w-16 h-full gap-1 text-gray-500 dark:text-gray-400 hover:text-gray-900 dark:hover:text-white transition-all active:scale-95 focus-visible:ring-2 focus-visible:ring-primary focus-visible:ring-offset-2 focus-visible:ring-offset-background-light dark:focus-visible:ring-offset-background-dark rounded-lg outline-none"
               aria-label="Open Notes"
           >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)